recipe-generator-ts-testing/tslint.json

```diff
@@ -0,0 +1,70 @@
+{
```

Comment:
You could put a `tslint.json` in the root of your project if you want to share the rules across all ts projects. You might still need the file here but it would just do `{parent: "../tslint.json"}` or something like that.

Reply:
good idea